src/vkscript/executor_test.cc

```diff
@@ -249,49 +279,212 @@ class VkScriptExecutorTest : public testing::Test {
   ~VkScriptExecutorTest() = default;
 
   std::unique_ptr<Engine> MakeEngine() { return MakeUnique<EngineStub>(); }
+  std::unique_ptr<Engine> MakeAndInitializeEngine(const Parser& parser) {
```

Comment:
Instead of passing the Parser, pass in the Features and Extension vectors.

Reply:
done